### ISSUE TITLE
Have you seen this model?

### ISSUE THREAD

**Issue by weswong** [edwenger/genepi#17]
Intra-host parasitemia model that reminds me a lot of DTK from David Fidock, Geoffrey Johnston, and David Smith

http://www.ncbi.nlm.nih.gov/pubmed/23637586

**Comment by edwenger** [comment]
Yes. My intention here was to take the EMOD DTK infectiousness as a function of time-since-infection (after making some approximation for population-level immunity and age-dependent biting risk).  But either the Fidock et al. or Swiss TPH formulation would be pretty appropriate to pull into the simpler genetics modeling as a modular sensitivity check...

**Comment by weswong** [comment]
awesome. I ran the default simulations (the ones fitted to the natural infection settings) to get some initiL equations that can be used to model asexual parasitemia over time. Then I will generate some coalescent distributions using ms to more Ccurately simulate mutation. I'm out for the next two weeks tho, so you may not hear back from me until July.
